robin88322/docker/example8/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y \
+    aufs-tools \
+    automake \
+    build-essential \
+    dpkg-sig \
+    libcap-dev \
+    libsqlite3-dev \
+    reprepro \
+    wget\
+ && rm -rf /var/lib/apt/lists/*
+
+RUN wget -O - https://github.com | wc -l > /number && cat /number
```

Comment:
What it the purpose of this image? Why don't you added ENTRYPOINT or CMD commands to output your number?